src/CodeGen_C.cpp

```diff
@@ -25,6 +25,7 @@ const string buffer_t_definition =
     "#ifndef BUFFER_T_DEFINED\n"
     "#define BUFFER_T_DEFINED\n"
     "#include <stdint.h>\n"
+    "#pragma pack(push, 1)\n"
```

Comment:
Is the pragma just for safety or is there really a platform that inserts extra space in this structure? (I understand that the padding is necessary to prevent some compilers from producing a sizeof that is 2 bytes less than others.)